react/features/base/participants/middleware.ts

```diff
@@ -241,7 +240,7 @@ MiddlewareRegistry.register(store => next => action => {
             const participant = getLocalParticipant(store.getState());
 
             if (participant && participant.id === action.id) {
-                APP.UI.emitEvent(UIEvents.NICKNAME_CHANGED, action.name);
+                store.dispatch(changeLocalDisplayName(action.name));
```

Comment:
Is this actually needed? If we are now using Redux we should be able to get rid of it, as per the TODO comment above.

Reply:
Yep. It actually turns out we don't dispatch this action at all. Will remove it.